core/drivers/bnxt/bnxt_fw.c

```diff
@@ -680,9 +680,11 @@ TEE_Result bnxt_load_fw(int chip_type)
 
 TEE_Result bnxt_copy_crash_dump(uint8_t *d, uint32_t offset, uint32_t len)
 {
+	size_t crash_len = 0;
```

Comment:
A better prefix for "core: add overflow check in bnxt_copy_crash_dump()" would be "drivers: bnxt:" IMO.

Reply:
Agree, I'll fix.